contracts/TokenConverter/RiskFundConverter.sol

```diff
@@ -28,6 +28,10 @@ contract RiskFundConverter is AbstractTokenConverter {
     /// @notice Address of pool registry contract
     address public poolRegistry;
 
+    /// @notice This mapping would contain the assets for the pool which would be send to RiskFund directly
+    /// @dev Comptroller(pool) -> Asset -> bool(should transfer directly on true)
+    mapping(address => mapping(address => bool)) public poolsAssetsDirectTransfer;
```

Comment:
We would need a setter function, integrated with the ACM. We would need to grant the Normal, Fast-track and Critical Timelocks to execute this function, in the VIP. And we have to emit an event in that new setter function, as usual

Reply:
46cd2cb1f80467a74f73cd78b00c5e61d729078a